src/sentry/static/sentry/app/views/organizationEvents/eventsChart.jsx

```diff
@@ -126,29 +126,27 @@ class EventsChart extends React.Component {
   }
 }
 
-const EventsChartContainer = withRouter(
-  withGlobalSelection(
-    withApi(
-      class EventsChartWithParams extends React.Component {
-        static propTypes = {
-          selection: SentryTypes.GlobalSelection,
-        };
-
-        render() {
-          const {selection, ...props} = this.props;
-          const {datetime, projects, environments} = selection;
-
-          return (
-            <EventsChart
-              {...datetime}
-              project={projects || []}
-              environment={environments || []}
-              {...props}
-            />
-          );
-        }
+const EventsChartContainer = withGlobalSelection(
+  withApi(
+    class EventsChartWithParams extends React.Component {
+      static propTypes = {
+        selection: SentryTypes.GlobalSelection,
+      };
+
+      render() {
+        const {selection, ...props} = this.props;
+        const {datetime, projects, environments} = selection;
+
+        return (
+          <EventsChart
```

Comment:
If EventsChart needs a router, where does this container get the router?

Reply:
@markstory 	It gets passed down from props

Reply:
Ah yeah since the `organizationEvents/events.jsx` component is part of a route, it gets passed `router` as a prop.